api/overview/azure/latest/event-hubs.md

```diff
@@ -6,43 +6,43 @@ ms.author: pallavit
 ms.data: 04/11/2023
 ms.topic: reference
 ms.devlang: dotnet
-ms.service: eventhubs
+ms.service: event-hubs

```

Comment:
Not sure where the font matter comes from.    @JimSuplizio, @danieljurek:  Can you assist?

Reply:
This file doesn't end with `-readme.md` . It's a service-level overview. Changes here should not be overwritten on publish.

Reply:
On closer inspection our service level readme automation does do special work here. Let me dig in more and see if my previous statement is still correct.

Reply:
The service level overview automation derives (or selects the `MSDocService`) from the first package it finds associated with that service. 

I recommend updating all package metadata for relevant service packages in the [metadata CSV file](https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv) to ensure that you have the right value @spelluru